app/views/teams/_form.html.slim

```diff
@@ -53,10 +53,9 @@
         Please make sure to add at least your team log feed and repositories that
         you are working on:
 
-    fieldset
+    fieldset id="team-sources"
```

Comment:
As said before, I'd suggest to switch to data attributes and put it directly on the radio button rather than this high up in the DOM.